### PR TITLE
Revert to monad aws style

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: antiope-core/src/Antiope/Core.hs [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+{-# LANGUAGE FlexibleInstances       #-}
+{-# LANGUAGE MultiParamTypeClasses   #-}
+{-# LANGUAGE RankNTypes              #-}
+{-# LANGUAGE ScopedTypeVariables     #-}
+{-# LANGUAGE TypeApplications        #-}
+{-# LANGUAGE TypeFamilies            #-}
+{-# LANGUAGE TypeInType              #-}
+{-# LANGUAGE TypeOperators           #-}
+{-# LANGUAGE UndecidableInstances    #-}
+{-# LANGUAGE UndecidableSuperClasses #-}
```
Comment (Kheldar) [inline antiope-core/src/Antiope/Core.hs]: Holy moly that's a lot of language extensions. ^_^

Comment (newhoggy) [reply antiope-core/src/Antiope/Core.hs]:
```
{-# LANGUAGE SuperSaiyan #-}
```

Achievement unlocked!

**Comment by Kheldar** [comment]
Works for me :)

**Comment by AlexeyRaga** [comment]
I don't like exposing `MonadAWS` constraints.

**Pull request closed without merging**